### PR TITLE
Added ALERT patterns to the CHP Lightbars

### PR DESCRIPTION
**Pull request by DatedGore**

### CODE REVIEW

File: lua/autorun/photon/library/auto/chp_whelen_liberty_sx.lua [hunk 1]
```diff
@@ -150,6 +150,11 @@ COMPONENT.Sections = {
 	["auto_whelen_liberty_ii_corner"] = {
 		[1] = { { 5, R, .55 }, { 7, B, .55 }, { 6, B, .55 }, { 8, R, .55 } }
 	},
+	["auto_whelen_liberty_alert"] = {
+		[1] = { { 2, B, .55 }, { 4, B, .55 }, { 5, R, .55 }, { 6, B, .55 }, 
+			{ 7, B, .55 }, { 8, R, .55 }, { 9, B, .55 }, { 10, R, .55 },  
+			{ 11, A, .55 }, { 12, A, .55 }, { 13, A, .55 }, { 14, A, .55 } }
+	}
```
Comment (JoshPiper) [inline lua/autorun/photon/library/auto/chp_whelen_liberty_sx.lua:157]: Why is that out as its own section, not in the sections which already exist?

File: lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua [hunk 2]
```diff
@@ -155,14 +150,19 @@ COMPONENT.Sections = {
...
 		[14] = {  { 11, A },  { 12, A },},
 		[15] = { },
 	},
 	["auto_whelen_liberty_ii_corner"] = {
 		[1] = { { 5, R, .55 }, { 7, B, .55 }, { 6, B, .55 }, { 8, R, .55 } }
+	},
+	["auto_whelen_liberty_alert"] = {
+		[1] = { { 2, B, .55 }, { 3, R, .55 }, { 4, B, .55 }, { 5, R, .55 }, 
+			{ 6, B, .55 }, { 7, B, .55 }, { 8, R, .55 }, { 9, B, .55 }, { 10, R, .55 },  
+			{ 11, A, .55 }, { 12, A, .55 }, { 13, A, .55 }, { 14, A, .55 } }
```
Comment (JoshPiper) [inline lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua]: As above, why is this in a new section?

Comment (DatedGore) [reply lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua]: wdym? The alert is its own section?

Comment (DatedGore) [reply lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua]: Its suppose to be like that.

Comment (DatedGore) [reply lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua]: ya have a problem with that?

Comment (DatedGore) [reply lua/autorun/photon/library/auto/whelen_liberty_ii_chp.lua]: I can change it if you want me to

**Review by JoshPiper — APPROVED** [review]
Works, no glaring issues I can see.

**Pull request merged**